auto_py_to_exe/web/css/general.css

```diff
@@ -58,13 +58,12 @@ input:focus {
 }
 
 button {
+    border: 2px solid #458BC6;
```

Comment:
The reason I didn't define the colour in here is because it's passed down from `button, input, select`, allows for consistency.

Reply:
When I was writing the new styles, I noticed that the `button, input, select` styles didn't change some parts of the ui. I think I can fix that though.